### DISCUSSION TITLE
https://imagekit.io/docs/api-keys?_gl=1*b9nmhe*_gcl_au*MTMyNTkxNzQzOC4xNzE5MzMxMjA3*_ga*MTgxMjA1MTI1Mi4xNzE5MzMxMjA2*_ga_WQGX7PLP73*MTcyMzU2Njk4OS40LjEuMTcyMzU2NzQ3OC41NS4wLjA.

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#54] [Docs]
# https://imagekit.io/docs/api-keys?_gl=1*b9nmhe*_gcl_au*MTMyNTkxNzQzOC4xNzE5MzMxMjA3*_ga*MTgxMjA1MTI1Mi4xNzE5MzMxMjA2*_ga_WQGX7PLP73*MTcyMzU2Njk4OS40LjEuMTcyMzU2NzQ3OC41NS4wLjA.

Learn about ImageKit's API keys, standard and restricted keys, and how to keep your keys safe.

https://imagekit.io/docs/api-keys?_gl=1*b9nmhe*_gcl_au*MTMyNTkxNzQzOC4xNzE5MzMxMjA3*_ga*MTgxMjA1MTI1Mi4xNzE5MzMxMjA2*_ga_WQGX7PLP73*MTcyMzU2Njk4OS40LjEuMTcyMzU2NzQ3OC41NS4wLjA.

**Comment by unlisted02** [comment]
I can reveal my Private Key, as I signed in using Google; I'm being prompted for password!! That's a huge bug.

**Reply by imagekitio** [reply]
This has been fixed now. A code is sent to your email regardless of how you signed up.
